lib/codex/widgets/codex_entry/components.dart

```diff
@@ -106,21 +104,22 @@ class _BuildBlueprint extends StatelessWidget {
   final List<Drop>? drops;
 
   void _onTap(BuildContext context) {
-    if (drops != null && (drops?.isNotEmpty ?? false)) {
-      Navigator.of(context).push(
-        MaterialPageRoute<void>(
-          builder: (context) => ComponentDrops(drops: drops!),
-        ),
-      );
-    }
+    Navigator.of(context).push(
+      MaterialPageRoute<void>(
+        builder: (context) => ComponentDrops(drops: drops!),
+      ),
+    );
   }
 
   @override
   Widget build(BuildContext context) {
+    final hasDrops = drops != null && (drops?.isNotEmpty ?? false);
+
     return InkWell(
-      onTap: () => _onTap(context),
+      onTap: hasDrops ? () => _onTap(context) : null,
       child: CircleAvatar(
         radius: 25,
+        // backgroundImage: CachedNetworkImageProvider(blueprintImage),
```

Comment:
no mo bg?

Reply:
I tried stretching it already...

![Screenshot_1716256010](https://github.com/WFCD/navis/assets/6075693/9edace9e-8e9e-4be3-bcba-9449b1b347e9)

Reply:
fair, this is a lot